src/models/affiliateModel.ts

```diff
@@ -0,0 +1,34 @@
+import {observable} from 'mobx';
+
+export class AffiliateModel {
+  @observable affiliateLink: string;
+  @observable referralsCount: number;
+  @observable totalTradeVolume: number;
+  @observable totalBonus: number;
+
+  constructor() {
+    this.affiliateLink = '';
```

Comment:
as a side note: you may also utilize es6 default assignments like this `private foo = 'bar'`